Simplenote/SPAppDelegate.m

```diff
@@ -420,6 +430,10 @@ - (void)setSelectedTag:(NSString *)selectedTag {
 
 - (void)bucket:(SPBucket *)bucket didChangeObjectForKey:(NSString *)key forChangeType:(SPBucketChangeType)change memberNames:(NSArray *)memberNames
 {
+    if (@available(iOS 13.0, *)) {
+        [self.refreshManager refreshTimer];
```

Comment:
I think it's better to name this method by saying what is happening instead of what should be done. 

For example if we say `[self.refreshManager onSimperiumObjectChange]`, `refreshManager` can decide what it wants to do in this case (and maybe it doesn't even want to `refreshTimer`)